.github/workflows/ci.yml

```diff
@@ -79,7 +81,7 @@ jobs:
         echo "${{ secrets.GITHUB_TOKEN  }}" | docker login docker.pkg.github.com -u awslabs --password-stdin
         export DOCKER_IMAGE=docker.pkg.github.com/awslabs/aws-crt-builder/aws-crt-${{ env.LINUX_BASE_IMAGE  }}:${{ env.BUILDER_VERSION  }}
         docker pull $DOCKER_IMAGE
-        docker run --env GITHUB_REF $DOCKER_IMAGE --version=${{ env.BUILDER_VERSION  }} build -p ${{ env.PACKAGE_NAME  }} --compiler=gcc-8 --spec=downstream cmake_args='-DBUILD_SHARED=ON'
+        docker run --env GITHUB_REF $DOCKER_IMAGE --version=${{ env.BUILDER_VERSION  }} build -p ${{ env.PACKAGE_NAME  }} --compiler=gcc-8 --spec=downstream 'cmake_args=-DBUILD_SHARED_LIBS=ON'
```

Comment:
I don't think the builder is handling cmake_args correctly yet.
When I "fixed" it the other day, I'd only tested in aws-c-common (which has no upstream dependencies).
When I tried it in aws-c-io, it didn't work.
I think you're hitting the same issue, where aws-c-common isn't getting built with -DBUILD_SHARED_LIBS=ON

Reply:
yes, correct. We should probably just add a `--shared` to builder that `CMakeBuild` picks up